guides/common/modules/con_prerequisites-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,45 @@
+[id="prerequisites-for-{project-context}-on-aws"]
+= Prerequisites for {Project} on AWS
+
+Installing and running {ProjectServer} and {SmartProxyServers} on Amazon Web Services (AWS) has additional requirements to your environment.
+
+* Ensure that the Amazon EC2 instance type meets or exceeds the {InstallingServerDocURL}system-requirements_{project-context}[System requirements] in _{InstallingServerDocTitle}_.
+For the best performance, use an https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html[AWS storage optimized instance].
```

Comment:
```suggestion
For the best performance, use an link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html[AWS storage optimized instance].
```
Or maybe even (nitpick trigger warning):
```suggestion
See also an link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html[AWS storage optimized instance] for further guidance.
```
Can we really promise that the guidelines will ensure "best" performance?